No rule to make target 'conv-autoconfig.h', needed by 'xi-main.o'

I'm attempting to build on alpine linux, using
```
build charm++ mpi-linux-x86_64 --build-shared --with-production --disable-charmdebug -j36 -O3 -DNDEBUG
```
I get
```
./configure: line 3806: syntax error: unexpected redirection
make[1]: *** No rule to make target 'conv-autoconfig.h', needed by 'xi-main.o'.  Stop.
make[1]: Leaving directory '/home/quinoa/quinoa-tpl/install/gnu-x86_64/charm/mpi-linux-x86_64/tmp'
```
Please let me know if you need more information.

--- a/src/scripts/configure.ac
+++ b/src/scripts/configure.ac
@@ -822,7 +822,7 @@ EOT
         got_opt_whole_archive="true"
         IFS=' ' read LDXX_WHOLE_ARCHIVE_PRE LDXX_WHOLE_ARCHIVE_POST <<EOF
         $i
-        EOF
+EOF
         add_flag "$(echo LDXX_WHOLE_ARCHIVE_PRE=\"$LDXX_WHOLE_ARCHIVE_PRE\")" "Flags to link whole archives into libcharm.so"
         add_flag "$(echo LDXX_WHOLE_ARCHIVE_POST=\"$LDXX_WHOLE_ARCHIVE_POST\")" "Flags to link whole archives into libcharm.so"
         break

--- a/src/scripts/configure.ac
+++ b/src/scripts/configure.ac
@@ -820,7 +820,9 @@ EOT
       if test $strictpass -eq 1
       then
         got_opt_whole_archive="true"
-        IFS=' ' read LDXX_WHOLE_ARCHIVE_PRE LDXX_WHOLE_ARCHIVE_POST <<< $i
+        IFS=' ' read LDXX_WHOLE_ARCHIVE_PRE LDXX_WHOLE_ARCHIVE_POST <<EOF
+        $i
+        EOF
         add_flag "$(echo LDXX_WHOLE_ARCHIVE_PRE=\"$LDXX_WHOLE_ARCHIVE_PRE\")" "Flags to link whole archives into libcharm.so"
         add_flag "$(echo LDXX_WHOLE_ARCHIVE_POST=\"$LDXX_WHOLE_ARCHIVE_POST\")" "Flags to link whole archives into libcharm.so"
         break

